rails-commenting.rb

```diff
@@ -6,57 +6,57 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) The controller is created in the Terminal, rails generate controller BlogPost would be the command. The controller allows for each of the CRUD routes to have a method. The 7 CRUD routes consist of Index, Show, Create, Edit, Update and Destroy. A model is passed through the controller, routes and views to establish an MVP which stands for minimum viable product.
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2) Index is a get request.This allows the user to see each instance of the Blogs that were created in the database. @post is passed in the erb file to output each blog. An example would be "@posts.each" with a do block to pass each post "do |blog|." By creating an index.html.erb file for this ruby method, the user can see each blog post in a list.
     @posts = BlogPost.all
```

Comment:
Exactly, this one is referencing that instance variable! Nice job talking about how it will be used to display all the blog posts.